conf/application.conf

```diff
@@ -158,7 +139,23 @@ microservice {
     }
 
     services {
-
+        letterAndControl {
```

Comment:
These should ideally be moved out of the microservice wrapper, but if this is needed to get it to work for now, go for it